src/libsystemd/sd-event/test-event.c

```diff
@@ -17,6 +17,8 @@
   along with systemd; If not, see <http://www.gnu.org/licenses/>.
 ***/
 
+#include <sys/wait.h>
```

Comment:
appears to be a muslism, or is there a definition we use that really really needs this header even on POSIX/glibc?